tutteli-gradle-kotlin-module-info/src/test/groovy/ch/tutteli/gradle/kotlin/module/info/ModuleInfoPluginIntTest.groovy

```diff
@@ -59,13 +67,40 @@ class ModuleInfoPluginIntTest {
         //not for jdk8
         assumeFalse(System.getProperty("java.version").startsWith("1.8"))
         //act
-        def result = moduleInfoInSubproject(settingsSetup, "requires kotlin.stdlib; requires ch.tutteli.atrium.bundle.cc.en_GB.robstoll;")
+        setupModuleInfoInSubproject(settingsSetup, "requires kotlin.stdlib; requires ch.tutteli.atrium.bundle.cc.en_GB.robstoll;")
+        def result = runGradleModuleBuild(settingsSetup, "sub1:jar")
         //assert
         Asserts.assertStatusOk(result, [':sub1:compileKotlin', ':sub1:compileModuleKotlin', ':sub1:compileModuleJava', ':sub1:inspectClassesForKotlinIC', ':sub1:jar'], [], [':sub1:classes'])
         assertTrue(result.output.contains("root has no sourceSets"), "root should not have sourceSets:\n$result.output")
         assertTrue(result.output.contains("sub1: source set 'module'"), "should contain sub1 source set 'module':\n$result.output")
     }
 
+    @Test
+    void compatibleToJava8(SettingsExtensionObject settingsSetup) {
+        //only for jdk8
+        assumeFalse(System.getProperty("java.version").startsWith("1.8"))
```

Comment:
should have been `assumeTrue` I guess

Reply:
No. It won’t work with a JDK 8 because that cannot compile `module-info.java`s. However, it *will* work with any JDK >= 9, because of the `sourceCompatability` and `targetCompatability` options.

Reply:
ok, so the comment was wrong, I have rewritten it so that the test works for jdk8 and newer